File: PAT (Basic-Level)/C/1038.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main()
{
    int N, K;
    //int count;
    scanf("%d", &N);
    int score[N];
    int grade[100 + 1] = {0};
    for(int i = 0; i < N; i++){
        scanf("%d", &score[i]);
        grade[score[i]]++;
    }

    scanf("%d", &K);
    int query[K];
    for(int j = 0; j < K; j++){
        scanf("%d", &query[j]);
    }

    for(int j = 0; j < K; j++){
        if(j < K - 1) printf("%d ", grade[query[j]]);
        else if (j == K - 1) printf("%d", grade[query[j]]);
    }
}

File: PAT (Basic-Level)/Python/1038.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

N = int(input())
score = list(map(int, input().split()))
query = list(map(int, input().split()))

for j in range(1, query[0]+1):
    count = 0
    for i in range(N):
        if query[j] == score[i]:
            count += 1
    if j < query[0]:
        print(count, end = ' ')
    elif j == query[0]:
        print(count, end = '')

#testpoint 3 overtime
